tests/test_ipa_nss.py

```diff
@@ -22,10 +24,7 @@ def make_group(name, members):
 
 
 class TestGroupMember(BaseTest):
-    @patch('grp.getgrnam')
-    def test_ipaapi_group_ok(self, mock_grp):
-        mock_grp.return_value = make_group('apache', ('apache', 'ipaapi',))
-
+    def test_ipaapi_group_ok(self):
```

Comment:
This assumes that the test system is configured properly doesn't it?

Reply:
Tests require `freeipa-server` RPM package which pulls `apache` as well, both have RPM post-scripts creating system users for them. Thus, IPA (including underlying components) has all the needed system users upon installation, the configuration is not required. So, actually mocking is required only for negative test cases. But adding a mocking for the existing group is not a big deal.